pkg/users/utils.js

```diff
@@ -12,3 +12,17 @@ export const get_locked = name =>
                     console.warn(`Failed to obtain account lock information for ${name}`, exc);
                 }
             });
+
+export async function getUtmpPath() {
+    try {
+        await cockpit.spawn(["test", "-e", "/var/run/utmp"], { err: "ignore" });
+        return "/var/run/utmp";
```

Comment:
Related, with the new systemd rc we now have a race condition with watching utmp so I want to change it to use logind's dbus API and listen to the relevant signals. (This does not have to block this PR but might help)

https://github.com/systemd/systemd/issues/32845

Reply:
We would like to get this included in the next release. If the changes for logind can be merged before than i think we can close this pr infavor of the logind pr